# goal
string name
string color
---
# result
bool flag_success
---
# feedback
int8 percentage_complete
